service/src/main/java/com/ge/predix/acs/audit/ACSAuditConfiguration.java

```diff
@@ -0,0 +1,87 @@
+package com.ge.predix.acs.audit;
+
+import java.util.List;
+
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+import com.ge.predix.audit.sdk.AuditCallback;
+import com.ge.predix.audit.sdk.AuditClient;
+import com.ge.predix.audit.sdk.AuditClientType;
+import com.ge.predix.audit.sdk.FailReport;
+import com.ge.predix.audit.sdk.config.AuditConfiguration;
+import com.ge.predix.audit.sdk.config.vcap.VcapLoaderServiceImpl;
+import com.ge.predix.audit.sdk.exception.AuditException;
+import com.ge.predix.audit.sdk.message.AuditEvent;
+import com.ge.predix.audit.sdk.validator.ValidatorReport;
+import com.ge.predix.eventhub.EventHubClientException;
+
+@Configuration
+@Profile("predixAudit")
+public class ACSAuditConfiguration {
+
+    private static final org.slf4j.Logger LOGGER = LoggerFactory.getLogger(ACSAuditConfiguration.class);
+
+    private VcapLoaderServiceImpl vcapLoaderService = new VcapLoaderServiceImpl();
+
+    private AuditConfiguration sdkConfig;
```

Comment:
Field can be converted to a local variable in the `auditClient` bean below (since it's only used there)